unitTests/ui/QuerySuggestPreviewTest.ts

```diff
@@ -194,6 +234,22 @@ export function QuerySuggestPreviewTest() {
         expect(test.cmp.queryController.getLastQuery().q).toBe(realQuery);
         done();
       });
+
+      it(`and the query get executed, 
+      an analytics get logs`, () => {
+        setupQuerySuggestPreview();
+        setupSuggestion();
+        triggerQuerySuggestHoverAndPassTime();
+        expect(test.cmp.usageAnalytics.logSearchEvent).toHaveBeenCalledWith(
+          analyticsActionCauseList.showQuerySuggestPreview,
+          jasmine.objectContaining({
+            partialQuery: last(omniboxAnalytics.partialQueries),
```

Comment:
The creation of this object is repeated on lines 82-85. It would be better to use a function to build and return the object.